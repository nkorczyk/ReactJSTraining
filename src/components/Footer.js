import React from 'react';

const Footer = () => {
  return (
    <div className="list2">
      <h5 className="red-text text-darken-2 grey darken-3 grey1">netflixroulette</h5>
    </div>
  )
}

export default Footer;
