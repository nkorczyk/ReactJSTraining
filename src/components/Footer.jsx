import React from 'react';
import CONSTANTS from '../constants/constants';

const Footer = () => {
  return (
    <div className="bar">
      <h5 className="red-text text-darken-2 grey darken-3 grey-bar">{CONSTANTS.NETFLIX}</h5>
    </div>
  )
}

export default Footer;
