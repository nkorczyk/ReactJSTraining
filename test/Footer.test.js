import React from 'react';
import { mount } from 'enzyme';
import Footer from '../src/components/Footer';

describe('should render Footer component', () => {
  const element = mount(<Footer />);
  it('Snapshot test with default props', () => {
    expect(element).toMatchSnapshot();
  });
});

it('should render netflixroulette string', () => {
  const element = mount(<Footer />);
  expect(element.text()).toEqual('netflixroulette');
});
