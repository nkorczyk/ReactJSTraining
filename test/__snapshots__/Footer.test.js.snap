// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer component Snapshot test with default props 1`] = `
<Footer>
  <footer
    className="bar"
  >
    <h5
      className="red-text text-darken-2 grey darken-3 grey-bar"
    >
      netflixroulette
    </h5>
  </footer>
</Footer>
`;
